pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1742,15 +1830,33 @@
       ::
       ++  event-16-to-17
         |=  events=(qeu queued-event-16)
-        ^-  (qeu queued-event)
+        ^-  (qeu queued-event-17)
         %-  ~(rep in events)
-        |=  [e=queued-event-16 q=(qeu queued-event)]
-        %-  ~(put to q)  ^-  queued-event
+        |=  [e=queued-event-16 q=(qeu queued-event-17)]
+        %-  ~(put to q)  ^-  queued-event-17
         ?.  ?=(%call -.e)  e
         =/  task=task-16  ((harden task-16) wrapped-task.e)
         %=  e
           wrapped-task  ?.(?=(%kroc -.task) task [%kroc ~])
         ==
+      ++  event-17-to-18
+        |=  events=(qeu queued-event-17)
+        ^-  (qeu queued-event)
+        %-  ~(rep in events)
+        |=  [e=queued-event-17 q=(qeu queued-event)]
+        %-  ~(put to q)  ^-  queued-event
+        ?.  ?=(%call -.e)  e
+        =/  task=task-17  ((harden task-17) wrapped-task.e)
+        %=  e
+          wrapped-task
+          ?.  ?=([%deep %nack *] task)  task
+          =/  msg  (cue message-blob.task)
```

Comment:
This should be in a `~+` (with a minimal subject).

Reply:
Done